gensim/corpora/__init__.py

```diff
@@ -3,15 +3,15 @@
 """
 
 # bring corpus classes directly into package namespace, to save some typing
-from .indexedcorpus import IndexedCorpus # must appear before the other classes
+from .indexedcorpus import IndexedCorpus  # noqa:F401 must appear before the other classes
 
-from .mmcorpus import MmCorpus
-from .bleicorpus import BleiCorpus
-from .svmlightcorpus import SvmLightCorpus
-from .lowcorpus import LowCorpus
-from .dictionary import Dictionary
-from .hashdictionary import HashDictionary
-from .wikicorpus import WikiCorpus
-from .textcorpus import TextCorpus
-from .ucicorpus import UciCorpus
-from .malletcorpus import MalletCorpus
+from .mmcorpus import MmCorpus  # noqa:F401
```

Comment:
-1: we don't want to litter the code with some tool-specific constructs.

Instead, open an issue with the tool that requires this, it's a bug on their side.

If you find such comments in other parts of gensim, please remove them.

Reply:
@piskvorky it's not a tool bug, it's really "unused import" by PEP

Reply:
But it is used -- it makes the classes available directly from the package. Without it, you won't be able to `from corpora import MmCorpus`.

Or how can this be done in other way, that doesn't trigger F401?

Reply:
> it makes the classes available directly from the package 
You are right, but now in the code, it's unused import.

I know another way - use this imports (like your example) in gensim internally, but I don't think that this is a good idea. For my opinion, `noqa` suppress is OK.